config.conf.example

```diff
@@ -24,3 +24,8 @@ AVAILABLE_LOCALES_CACHE_EXPIRY = 24
 # Localisation replacement strings, see assets/strings/*.json for usage:
 [REPLACEMENTS]
 maintainer = <a href='http://wiki.apertium.org/wiki/Apertium' target='_blank'>Apertium</a>
+
+[SUGGESTIONS]
+ENABLED = True
+RECAPTCHA_SITE_KEY = None
+CONTEXT_WRAP = 5
```

Comment:
Why is this called `CONTEXT_WRAP`? I can think of more sensible things to call it :)